src/app/client/src/app/modules/dashboard/components/filter/filter.component.html

```diff
@@ -0,0 +1,87 @@
+  <form [formGroup]="filtersFormGroup" class="ui form"
+  [ngClass]="{'hide': hideElements}" *ngIf="showFilters && !hideElements">
+  <div class="d-flex flex-w-wrap reset-filter-section">
+    <div *ngIf="showGraphStats" class="d-flex flex-ai-center">{{resourceService?.frmelmnts?.lbl?.graphStat }}:
+      <sui-checkbox class="toggle ml-8" appTelemetryInteract [telemetryInteractObject]="telemetryInteractObject"
+        [telemetryInteractEdata]="setTelemetryInteractEdata('graph-statistics')"
+        [telemetryInteractCdata]="telemetryCdata" (checkChange)="statsChange($event)"></sui-checkbox>
+    </div>
+    <div class="mr-8" *ngIf="chartType === 'bar' || chartType === 'line'">
+      <!-- <button (click)="addChartSummary()" *ngIf="chartConfig.id && isUserReportAdmin" type="button"
```

Comment:
Why is this piece of code commented out. It's for adding a chart summary...

Reply:
this "addChartSummary" function handled in data-chart.component.html